gm4_block_compressors/data/gm4_block_compressors/functions/compress/16.mcfunction

```diff
@@ -8,7 +8,7 @@ data modify entity @s Item.tag.gm4_precompression_tag set from entity @s Item.ta
 execute unless data entity @s Item.tag run data modify entity @s Item.tag.gm4_precompression_tag set value "no_tag_data_to_preserve"
 
 #add new data to indicate store compression state
-data merge entity @s {Item:{Count:1b,tag:{CustomModelData:16,gm4_compressed:{value:16},HideFlags:1,Enchantments:[{id:"minecraft:protection",lvl:0s}]}}}
+data merge entity @s {Item:{Count:1b,tag:{CustomModelData:16,gm4_compressed:{value:16},HideFlags:1,Enchantments:[{}]}}}
```

Comment:
This change will break compressed items as empty Enchantments entries are deleted by some server implementations. The standard way of avoiding this is by the use of a level 0 enchant, such as protection (as implemented before your changes).

Reply:
Mojang also seemingly keeps alternating whether or not an empty enchantments list will give the item the glint or not. An actual hidden enchantment will always work.